tests/Unit/Domain/Test_DomainHelpers.cpp

```diff
@@ -38,8 +38,8 @@
 #include "tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp"
 
 namespace domain {
-SPECTRE_TEST_CASE("Unit.Domain.DomainHelpers.Periodic.SameBlock",
-                  "[Domain][Unit]") {
+namespace {
```

Comment:
FYI @nilsdeppe is doing something similar  in #1996 (also in a separate commit)

Reply:
Dropped the commit in #1996